srcpkgs/python3-notebook_shim/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'python3-notebook_shim'
+pkgname=python3-notebook_shim
+version=0.2.2
+revision=1
+build_style=python3-pep517
+make_install_target="dist/notebook_shim-${version}-*-*-*.whl"
+hostmakedepends="hatchling"
+depends="python3-jupyter_server"
+checkdepends="python3-pytest python3-jupyter_server python3-platformdirs"
+short_desc="Shim layer for notebook traits and config"
+maintainer="dkwo <npiazza@disroot.org>"
+license="BSD-3-Clause"
+homepage="https://github.com/jupyter/notebook_shim"
+distfiles="https://github.com/jupyter/notebook_shim/releases/download/v${version}/notebook_shim-${version}.tar.gz"
```

Comment:
We prefer `${PYPI_SITE}/n/notebook_shim/notebook_shim-${version}.tar.gz` if it exists.

Reply:
ok